nmtwizard/framework.py

```diff
@@ -234,6 +234,8 @@ def declare_arguments(self, parser):
         parser_preprocess = subparsers.add_parser('preprocess', help='Sample and preprocess corpus.')
         parser_preprocess.add_argument('--build_model', default=False, action='store_true',
                                        help='Preprocess data into a model.')
+        parser_preprocess.add_argument('-o', '--output', help='Output files')
```

Comment:
This argument will be ignored if `build_model` argument is set. Maybe we should add some explicit error when both are set ?